romana/cmd/host.go

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2016 Pani Networks
+// All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/romana/core/common"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// hostCmd represents the host commands
```

Comment:
Yes, but... what does that mean? Remember that I didn't even have a package comment yet, so I have no idea what this whole bit here is about. So at this point, this docstring doesn't tell me anything. What is a 'host command'? Why do we need to represent it? Etc.